content/en/blog/_posts/YYYY-MM-DD-Call-to-Action-Survey-of-Kubernetes-Tool-Preferences-for-Programming-Tasks.md

```diff
@@ -0,0 +1,24 @@
+title: "Call to Action: Survey of Kubernetes Tool Preferences for Programming Tasks"
+date: 2021-10-29
+slug: call-to-action-survey-of-kubernetes-tool-preferences-for-programming-tasks
+---
+
+**Authors:** Cora Coleman (UCSD) and Nick Mitchell (IBM)
+
+Understanding the software tools that are preferred by Kubernetes developers and whether preference depends on the programming task at hand, is important for informing the future design and development of these tools. To help understand how people with different levels of development experience are using and prefer to use Kubernetes, we are currently performing a 2-minute [Kubernetes Tasks and Tools survey](https://forms.gle/CcXwjx1Tftkp7KAW9) that you can still take! The survey asks whether you are a beginner, intermediate, or expert Kubernetes developer and for your preference among three tool modalities: Command Line Interface (CLI), Graphical Console, and IDE, to examine whether your tool preference varies across three programming task types: CRUD, Debugging, and Monitoring.
+
+The responses are currently coming in! Thank you to all those who have already contributed to the survey. Our goal is to make the raw data results open and available for everyone once more responses are collected.
+
+## The Research Gap
+
+Developer surveys such as the [CNCF](https://www.cncf.io/wp-content/uploads/2020/11/CNCF_Survey_Report_2020.pdf) show that there is an increased usage of Kubernetes in production. The [Apps SIG](https://github.com/kubernetes/community/tree/master/sig-apps) [survey](https://kubernetes.io/blog/2018/04/24/kubernetes-application-survey-results-2018/) reports on which Kubernetes tools are preferred in practice. However, these surveys do not investigate the combined impact of different tool modalities and programming task types on preference. Our survey seeks to fill this gap and build on these findings by reporting on people's level of Kubernetes experience and asking about the context in which their preferred tools are used. Understanding what programming task a preferred tool is used to accomplish can help to illuminate when the tool is the most helpful and what features of the tool contribute to this. 
+
+## You Can Help
+
+Good candidates for participating in this survey include those that have any Kubernetes experience, anywhere from a beginner to an expert level. For beginners, even if you've only been using Kubernetes for a few weeks, we'd appreciate your feedback. We welcome participants with experience using any form of Kubernetes tools. Furthermore, we encourage others involved within open source Special Interest Groups (SIGs) for [CLIs](https://github.com/kubernetes/community/blob/master/sig-cli/README.md) and [Usability](https://github.com/kubernetes-sigs/sig-usability) to share the survey within the Kubernetes community. We are still recruiting survey participants with any experience using Kubernetes in their work.
+
+## [Take Our Survey](https://forms.gle/CcXwjx1Tftkp7KAW9)
```

Comment:
I wouldn't put the link in the header. Please instead repeat it in the running text so it's obvious that readers should click on it.

Reply:
Will put the link in the running text and not the header.